package/network/services/samba36/files/samba.init

```diff
@@ -109,10 +108,12 @@ start_service() {
 	procd_open_instance
 	procd_set_param command /usr/sbin/smbd -F
 	procd_set_param respawn
+	procd_set_param file /var/etc/smb.conf
 	procd_close_instance
```

Comment:
This line is already in the description:
* Force service stop/start upon config change

As far as I understand, "procd_set_param file" triggers an explicit restart (stop/start) when reload is called only if the file is changed. I'd be happy to change the description to something better - you want me to mention the function?